controller_manager:
  ros__parameters:
    update_rate: 50

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


joint_trajectory_controller:
  ros__parameters:
    joints:
      - wrist_1_joint
      - wrist_2_joint
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
